schema.sql

```diff
@@ -9,4 +9,26 @@ CREATE TABLE animal (
     weight_kg decimal
 );
 
-Alter TABLE animal ADD species char(20);
\ No newline at end of file
+Alter TABLE animal ADD species char(20);
+
+
+CREATE TABLE owners (
+    id int GENERATED BY DEFAULT AS IDENTITY,
+    full_name varchar(100),
+    age decimal,
```

Comment:
- Please, kindly ensure the age is an integer and not a decimal to meet the project requirements.